### ISSUE TITLE
Upsampling Issue

### ISSUE THREAD

**Issue by krishnakarthi** [bonlime/keras-deeplab-v3-plus#131]
Hi,
I'm using Deeplab v3 keras based model to train my custom dataset from scratch, here my image size is 512 x 512. When i look upsampling architecture in the Lambda layer its suddenly increasing from 64 to 512. Can you please give me reason for this sudden increase from 64 to 512 or am i missing here.

![image](https://user-images.githubusercontent.com/46548656/71449066-d82c7e80-276b-11ea-80fb-4d0dd978d9d4.png)

dropout_1 (Dropout)             (None, 64, 64, 256)  0           activation_2[0][0]               
__________________________________________________________________________________________________
custom_logits_semantic (Conv2D) (None, 64, 64, 10)   2570        dropout_1[0][0]                  
__________________________________________________________________________________________________
lambda_4 (Lambda)               (None, 512, 512, 10) 0           custom_logits_semantic[0][0]     
__________________________________________________________________________________________________
reshape_1 (Reshape)             (None, 262144, 10)   0           lambda_4[0][0]                   
__________________________________________________________________________________________________

**Comment by zainulabidin302** [comment]
@krishnakarthi , looks like the issue is coming due to difference in "input_shape" and "input_tensor" parameters.
Are you passing both?
